### ISSUE TITLE
Plotly-plot of the components

### ISSUE THREAD

**Issue by rednafi** [facebook/prophet#1023]
Please add **plotly plot** options for **component plotting** `m.plot_components `. Currently, as of version 0.5 `plot_plotly` doesn't support that. 

So the main plot looks sleek while the component plot (plotted with `matplotlib` looks dull)
![Screenshot from 2019-06-19 13-45-34](https://user-images.githubusercontent.com/30027932/59746799-73163e80-9299-11e9-9ef0-08699978e3d7.png)
![Screenshot from 2019-06-19 13-45-25](https://user-images.githubusercontent.com/30027932/59746801-73163e80-9299-11e9-877e-f29510fd5d0d.png)

**Comment by bletham** [comment]
That would be a great new feature!

**Comment by bletham** [comment]
Done in #1038 by @Olof-Hojvall

**Comment by bletham** [comment]
I'll leave this open until we actually push it to PyPI.

**Comment by magnawhale** [comment]
So I'm fairly new to using FBProphet, so I apologize if I've misinterpreted anything about how the libraries work. 
I tried to get the components plotted using the plot.ly functions provided here: https://github.com/facebook/prophet/blob/master/python/fbprophet/plot.py
However, I kept getting an error message when trying to plot the components. As far as I can tell, the provided code is incorrect. There are two line that read like this:

`yaxis = go.layout.YAxis(rangemode='normal' if name == 'trend' else 'tozero',
                            title=go.layout.yaxis.Title(text=name),
                            zerolinecolor=zeroline_color)`

but to get them working, both of them must be changed to this:

`yaxis = go.layout.YAxis(rangemode='normal' if name == 'trend' else 'tozero',
                            title=name,
                            zerolinecolor=zeroline_color)`
